provision.sh

```diff
@@ -82,14 +90,15 @@ sudo virsh pool-autostart default
 
 echo "Installing oc client"
 cd $HOME
-curl -OL https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.1.8/openshift-client-linux-4.1.8.tar.gz
-tar -zxf openshift-client-linux-4.1.8.tar.gz
-rm -fr openshift-client-linux-4.1.8.tar.gz
-sudo mv $HOME/oc $HOME/kubectl /usr/local/bin
+curl -OL https://mirror.openshift.com/pub/openshift-v4/clients/oc/4.4/linux/oc.tar.gz
```

Comment:
Shouldn't we just use https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/ since we are going to run tests on the installer master?

Reply:
yes, i'll update that!